alicloud/table_alicloud_ecs_instance.go

```diff
@@ -425,6 +425,13 @@ func tableAlicloudEcsInstance(ctx context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.FromField("PublicIpAddress.IpAddress"),
 			},
+			{
+				Name:        "ram_role",
+				Description: "The public IP addresses of instances.",
```

Comment:
The description seems a bit of a misfit for the column name. Are we using the correct description?